src/SFA.DAS.FAT.Web.UnitTests/Controllers/CoursesControllerTests/WhenGettingCourseProviderDetails.cs

```diff
@@ -41,6 +43,10 @@ public class WhenGettingCourseProviderDetails
             Assert.IsNotNull(actualResult);
             var actualModel = actualResult.Model as CourseProviderViewModel;
             Assert.IsNotNull(actualModel);
+            var actualModelAdditionalCourses = actualModel.AdditionalCourses as ProviderCoursesViewModel;
+            Assert.IsNotNull(actualModelAdditionalCourses);
+            var actualModelAdditionalCoursesList = actualModel.AdditionalCourses.Courses as List<ProviderCourseViewModel>;
+            Assert.IsNotNull(actualModelAdditionalCoursesList);
```

Comment:
This can just be simplified to 
`Assert.IsNotNull(actualModel.AdditionalCourses.Courses);`
You don't need to do the type cast